static/game/statuseffects/burning.js

```diff
@@ -39,9 +39,10 @@ export class Burning extends StatusEffect {
       } else {
         color = "hsl(" + Math.floor(Math.random() * 55 - 5) + ", 100%, 50%)";
       }
-      const xOffset = Math.random() * creature.width - (creature.width / 2);
-      const yOffset = Math.random() * creature.height - (creature.height / 2);
+      const xOffset = Math.random() * creature.width - creature.width / 2;
+      const yOffset = Math.random() * creature.height - creature.height / 2;
```

Comment:
These are things that Prettier fixed when I saved the file. We should probably have it enforced somehow so everyone is using it.